mrp_stock_move_actual_date/models/mrp_production.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MrpProduction(models.Model):
+    _name = "mrp.production"
+    _inherit = ["mrp.production", "actual.date.mixin"]
+
+    def _get_actual_date_update_triggers(self):
+        return super()._get_actual_date_update_triggers() + [
+            "date_done",
+            "move_raw_ids",
```

Comment:
@AungKoKoLin1997 Please confirm the validity of this suggestion.
```suggestion
            "date_finished", "move_raw_ids", "move_finished_ids", "move_byproduct_ids"
```

Reply:
I don't see the case we need to trigger when `date_finished` is updated because there is no `actual_date` value before installing this module. Then, we don't need to add `move_byproduct_ids` because `move_finished_ids` will also link with this stock move and `move_byproduct_ids` is linked with the same stock move of `move_finished_ids` when `move_finished_ids` is updated. So, just adding `move_finished_ids` is enough.

Reply:
@AungKoKoLin1997 Thanks for checking. You are right about `move_byproduct_ids` since it's just a filtered subset of `move_finished_ids`. However, I disagree with `date_finished`. We didn't add `date_done` as a trigger in `stock_move_actual_date` just because we have cases of `actual_date` already populated before the update (it's not the right approach if that was the sole reason). The trigger was added for extra security to make sure that `actual_date` is updated in related stock moves when the record completes. The same should be applied to the production order.

Reply:
I agree.